.travis.yml

```diff
@@ -1,24 +1,30 @@
 language: julia
+
 os:
   - linux
-  - osx
+#  - osx
+
 julia:
   - 1.0
   - nightly
+
 matrix:
   allow_failures:
     - julia: nightly
   fast_finish: true
+
 notifications:
   email: false
-script:
-  - if [[ -a .git/shallow ]]; then git fetch --unshallow; fi
-  - julia -e "
-        using Pkg;
-        Pkg.clone(pwd(), \"MCMCChain\");
-        Pkg.build(\"MCMCChain\");
-        Pkg.test(\"MCMCChain\"; coverage=true)
-      "
+
+#script:
```

Comment:
any reason for removing Travis tests?

Reply:
I uncommented these lines as it is sufficient and probably better to run the default Travis script for Julia packages. We do not need any specialized code here and using the default removes the burden of maintaining the Travis file.